client/nginx.go

```diff
@@ -193,6 +196,20 @@ type NginxInfo struct {
 	ParentProcessID uint64 `json:"ppid"`
 }
 
+// LicenseReporting contains information about license status for NGINX Plus.
+type LicenseReporting struct {
+	Healthy bool
+	Fails   uint64
+	Grace   uint64
+}
+
+// NginxLicense contains licensing information about NGINX Plus.
+type NginxLicense struct {
+	ActiveTill uint64 `json:"active_till"`
```

Comment:
What is "till"?

Reply:
ActiveTill is a unix epoch

Reply:
It represents when the license expires.  `active_till` is what it is called in the nginx plus api.